pkg/tracer/event.go

```diff
@@ -39,9 +39,15 @@ func tcpV4ToGo(data *[]byte) (ret TcpV4) {
 	return
 }
 
+// Value added to all timestamps, to hold back events so they are less
+// likely to be reported out of order
+var (
+	timestampOffset uint64
```

Comment:
I suggest exporting that and giving it a sensible default. That way we don't need `SetTimestampOffset()` (which is a strange creature anyway, since it's a method that sets a global), and callers don't need to do anything unless they are unhappy with the default.

Reply:
Done.